modules/light-clients/06-solomachine/types/misbehaviour.go

```diff
@@ -10,48 +10,48 @@ import (
 	"github.com/cosmos/ibc-go/v3/modules/core/exported"
 )
 
-var _ exported.Misbehaviour = &Misbehaviour{}
+var _ exported.Misbehaviour = &DuplicateSignatures{}
```

Comment:
I didn't rename the flies. I figure if the naming for `Misbehaviour` interface is kept then it's probably fine, otherwise I can update the file name appropriately to something like `duplicate_signatures.go`. cc. @colin-axner

Reply:
we can do it in a follow up if we so choose